packages/sources/src/components/tile-grid/tile-grid.scss

```diff
@@ -0,0 +1,7 @@
+@mixin tile-grid_root() {
+  display: grid;
+  gap: $u4;
```

Comment:
Er is een variable voor de gutter width, ik denk dat we die hier ook willen inzetten.

Reply:
En toen ik dit dit wijzigde zag ik dat er een `margin: $u2` werd gezet, waarvan ik al dacht.. dat zou zomaar eens de helft van de gutter kunnen zijn, en dat je hiermee het responsive grid emuleert. En jawel hoor:

![image](https://user-images.githubusercontent.com/9696261/161630327-eceb3204-b391-4a47-bc32-7ba5976bf2ce.png)

Ik ben zo vrij geweest om 'm eraf te halen. De volledige truuk is: negative margin + positieve padding. Met CSS grid is deze truuk niet meer nodig, dus heb ik 'm eraf gehaald.

Daarnaast vond ik dat de totale whitespace tussen de tiles en de inhoud van Viewer Grid ook wat aan de forse kant was.